src/software/ai/hl/stp/tactic/receiver/receiver_tactic.h

```diff
@@ -16,7 +16,7 @@
 class ReceiverTactic : public Tactic
 {
    public:
-    ReceiverTactic();
+    explicit ReceiverTactic(const TbotsProto::ReceiverTacticConfig& receiver_config);
```

Comment:
docs

Reply:
bump

Reply:
New gameplay will change this constructor to take a `Strategy` so we can skip writing docs for this